crates/ruff_server/src/format.rs

```diff
@@ -0,0 +1,29 @@
+use ruff_formatter::PrintedRange;
+use ruff_python_formatter::{format_module_source, format_range};
+use ruff_text_size::TextRange;
+use ruff_workspace::FormatterSettings;
+
+use crate::edit::Document;
+
+pub(crate) fn format(
+    document: &Document,
+    formatter_settings: &FormatterSettings,
+) -> crate::Result<String> {
+    // TODO(jane): support Jupyter Notebook
+    let format_options = formatter_settings
+        .to_format_options(ruff_python_ast::PySourceType::Python, document.contents());
+    let formatted = format_module_source(document.contents(), format_options)?;
+    Ok(formatted.into_code())
+}
+
+pub(crate) fn range_format(
```

Comment:
I would call the method `format_range` and instead use the full qualified name on line 28. I fear that it's otherwise confusing because people need to remember whether it was `format_range` or `range_format` and the answer depends on the crate you're in.